src/app/views/login/SignInForm.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import Input from "../../components/Input";
+
+const propTypes = {
+    formData: PropTypes.shape({
+        inputs: PropTypes.arrayOf(
+            PropTypes.shape({
+                id: PropTypes.string,
+                label: PropTypes.string,
+                type: PropTypes.string,
+                onChange: PropTypes.func,
+                error: PropTypes.string
+            })
+        ),
+        onSubmit: PropTypes.func,
+        isSubmitting: PropTypes.bool
+    })
+};
+
+export default class LoginForm extends Component {
```

Comment:
As we're only really working with props and presentational elements here, this could be a good candidate for refactoring into a functional component. something like:

```js
// or export default ({ // }) => etc.
const LoginForm = ({ inputs, isSubmitting, onSubmit }) => {
    return (
        // markup
    )
}
```

This is quite a specific component so won't be overly reused I suspect, but I think destructuring the props would make it clearer as well what exactly is required, and also reduce the need to set new variables to abstract away the `this.props.formData` reference